postgres/lib/connect.js

```diff
@@ -144,6 +162,8 @@ function create(pool, parentCache) {
 					WHERE 
    					c.table_name = $1 and c.table_schema = 'public';
 			`, [table], (err, results) => {
+				if (results.length === 0) noTable = true;
+				if (err) throw err;
```

Comment:
This should not throw an error, it should emit an error
function(){
   if (err){
      return theStream.emit("error", err);
   }
}
let theStream = ls.bufferBackoff().....